### ISSUE TITLE
Error with bin/behat

### ISSUE THREAD

**Issue by vikey89** [Sylius/Sylius#4049]
Hi guys,
I'm on the master branch synced with the last commit and with Symfony versions 2.8.2. 
When I launch the command `bin/behat`, I get this error:

````
PHP Fatal error:  Call to private appTestDebugProjectContainer::__clone() from context 'Sylius\Behat\PageObjectExtension\ServiceContainer\PageObjectExtension' in /var/www/Sylius/src/Sylius/Behat/PageObjectExtension/ServiceContainer/PageObjectExtension.php on line 99

PHP Stack trace:
PHP   1. {main}() /var/www/Sylius/vendor/behat/behat/bin/behat:0
PHP   2. Symfony\Component\Console\Application->run() /var/www/Sylius/vendor/behat/behat/bin/behat:31
PHP   3. Behat\Testwork\Cli\Application->doRun() /var/www/Sylius/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:123
PHP   4. Behat\Testwork\Cli\Application->createCommand() /var/www/Sylius/vendor/behat/behat/src/Behat/Testwork/Cli/Application.php:96
PHP   5. Behat\Testwork\Cli\Application->createContainer() /var/www/Sylius/vendor/behat/behat/src/Behat/Testwork/Cli/Application.php:138
PHP   6. Symfony\Component\DependencyInjection\ContainerBuilder->compile() /var/www/Sylius/vendor/behat/behat/src/Behat/Testwork/Cli/Application.php:164
PHP   7. Symfony\Component\DependencyInjection\Compiler\Compiler->compile() /var/www/Sylius/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:589
PHP   8. Sylius\Behat\PageObjectExtension\ServiceContainer\PageObjectExtension->process() /var/www/Sylius/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/Compiler.php:107
PHP   9. Sylius\Behat\PageObjectExtension\ServiceContainer\PageObjectExtension->extractSymfonyApplicationRouter() /var/www/Sylius/src/Sylius/Behat/PageObjectExtension/ServiceContainer/PageObjectExtension.php:63
PHP  10. Sylius\Behat\PageObjectExtension\ServiceContainer\PageObjectExtension->extractSymfonyApplicationContainer() /var/www/Sylius/src/Sylius/Behat/PageObjectExtension/ServiceContainer/PageObjectExtension.php:86
```

Any idea?

**Comment by steffenbrem** [comment]
Have you cleared your cache folder?

**Comment by vikey89** [comment]
yes, I run `php app/console cache:clear`

**Comment by steffenbrem** [comment]
But did you do that on the test environment too? What I normally do is clear everything in app/cache, just to make sure.

**Comment by steffenbrem** [comment]
Forgot to mention, but you can use the --env flag to set the target environment to run the command against. So `php app/console cache:clear --env test` would clear your test cache.

**Comment by vikey89** [comment]
I already tried that step, I suspect the problem is in the symfony version.

**Comment by steffenbrem** [comment]
@vikey89 It's related to this specific commit https://github.com/symfony/symfony/commit/ba129041ba19b6d5a62cfa34789d3b05e6efbacf

**Comment by steffenbrem** [comment]
@pjedrzejewski I am not sure if this is a problem related to Sylius or not? Should Sylius support `2.8`? If so, we should add cloning support to the test container in the test kernel.

**Comment by vikey89** [comment]
thanks @steffenbrem for you suggestion, I had guessed that the problem was Symfony. I will try to back at version 2.7.7 of Symfony and I will wait the answer by @pjedrzejewski

**Comment by michalmarcinkowski** [comment]
@steffenbrem @vikey89 we want to be compatible with Symfony 2.8 since they dropped support for cloning container we should somehow work around this. 

Thanks for reporting! :+1:

**Comment by pamil** [comment]
@vikey89 it will be solved in a PR I'll open today

**Comment by vikey89** [comment]
ok @pamil thanks :)

**Comment by pamil** [comment]
@vikey89 can you check if this issue was solved by #4053?

**Comment by vikey89** [comment]
@pamil yes it works, thanks :+1:
